Mu2eG4/geom/ExtShieldUpstream_v07.txt

```diff
@@ -0,0 +1,206 @@
+vector<double> ExtShieldUpstream.centerType1Box5 = {812.5,-382,2535.2};//BRP152
+vector<double> ExtShieldUpstream.centerType2Box1 = {812.5,-382,1659.2};//BRQ152
+vector<double> ExtShieldUpstream.centerType2Box2 = {1345.5,-382,2827.2};//BRQ152
+vector<double> ExtShieldUpstream.centerType2Box3 = {1878.5,-382,1761.2};//BRQ152
+vector<double> ExtShieldUpstream.centerType3Box1 = {1802.5,-382,-1661.5};//BRR-152
+vector<double> ExtShieldUpstream.centerType4Box1 = {1258.5,-636,-3776.5};//BRR-132
+vector<double> ExtShieldUpstream.centerType5Box1 = {1052.5,1840,-223.8};//BRN-264
+vector<double> ExtShieldUpstream.centerType6Box1 = {127,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType6Box2 = {-711,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType6Box3 = {-1549,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType6Box4 = {-2387,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType6Box5 = {-3225,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType6Box6 = {-4063,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType6Box7 = {-4901,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType6Box8 = {-5739,-90,-3087};//BRW-175
+vector<double> ExtShieldUpstream.centerType7Box1 = {1052.5,2488,-238.5};//BRN-264
+vector<double> ExtShieldUpstream.centerType7Box2 = {1047.5,3199,-238.5};//BRN-264
+vector<double> ExtShieldUpstream.centerType8Box1 = {-2781.5,2488,-3087};//West
+vector<double> ExtShieldUpstream.centerType8Box2 = {-2781.5,3199,-3087};//Caps
+vector<double> ExtShieldUpstream.centerType9Box1 = {5261.4,237.3,-600.5};//protection collimator
+vector<double> ExtShieldUpstream.centerType10Box1 = {1345.5,-382,1951.2};//3
+vector<double> ExtShieldUpstream.centerType10Box2 = {2196,-382,2319.7};//BRP152
+vector<double> ExtShieldUpstream.centerType10Box3 = {2196,-382,2852.7};//BRP152
+vector<double> ExtShieldUpstream.centerType11Box1 = {1878.5,-382,-2550};//BRP152
+vector<double> ExtShieldUpstream.centerType11Box2 = {2411.5,-382,-2550};//BRP152
+// Special block to shield CR muons
+vector<double> ExtShieldUpstream.centerType12Box1 = {-2552.8,-381.4, 2413.0};
+vector<double> ExtShieldUpstream.centerType13Box1 = {-1473.2,-381.4, 1222.4};
+vector<double> ExtShieldUpstream.centerType14Box1 = {0.0,-381.4, -1186.1};
+// Borated poly
+vector<double> ExtShieldUpstream.centerType15Box1 = {-3018.2,1454.5,-2524.7};
+vector<double> ExtShieldUpstream.centerType16Box1 = {-2655.4,1175.1,-2260.6};
+vector<double> ExtShieldUpstream.centerType17Box1 = {-2909.4,2526.2,  203.2};
+vector<double> ExtShieldUpstream.centerType18Box1 = {-3620.6,2780.2,  213.6};
+vector<double> ExtShieldUpstream.centerType19Box1 = {-267.8 ,2952.0,  202.4};
+
+// Remote handling room door
+vector<double> ExtShieldUpstream.centerType20Box1 = {9381 ,-103, -11931};
+
+
+//---------------------------------Remote Handling Door---------------------------------
```

Comment:
This is a small enough change that I recommend just adding to ExtShieldUpstream_v06.txt instead of creating a new file.  

This will mean that you do not need changes to any of the geom_2021_PhaseI* files or to geom_MARS_2019.txt

Reply:
alright.